### ISSUE TITLE
Unknown character displaying on .INI document

### ISSUE THREAD

**Issue by westor7** [notepad-plus-plus/notepad-plus-plus#5440]
### Description of the Issue
I created a code to write down a .ini file but when i open it to see what is going on i saw that in notepad++ is displaying an extra character "Â" there somewhere but i've never request that character to write, i compare it with MS notepad and i saw that is an issue.

### Steps to Reproduce the Issue
1. Create a .ini file and paste this raw https://pastebin.com/sebc0YVs then save it and open it via Notepad++ there is special extra character displayed there as it shouldn't

### Expected Behavior
To not display the "Â" character there because i've never added.

### Actual Behavior
It's displaying an extra (never written) character there "Â" 

### Debug Information
Notepad++ v7.6.4   (64-bit)
Build time : Mar  6 2019 - 02:58:24
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : DSpellCheck.dll mimeTools.dll NppConverter.dll 

Screenshot differences between notepad.exe and notepad++.exe ---> https://i.imgur.com/MMdqPV8.png

**Comment by sasumner** [comment]
Why don't you look at your file in a hex editor (e.g. hxd)?  (Not Notepad++'s hex editor plugin)

A hex editor won't lie to you.

**Comment by MetaChuh** [comment]
**Referenced to:**
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/bfb7e863cc6cd20f9f32389bee289607a212b12b , https://github.com/notepad-plus-plus/notepad-plus-plus/issues/5299 , https://github.com/notepad-plus-plus/notepad-plus-plus/issues/5310 , https://github.com/notepad-plus-plus/notepad-plus-plus/issues/5202 , https://github.com/notepad-plus-plus/notepad-plus-plus/pull/5414 

@westor7  

my apologies for your confirmed and verified inconvenience.

unfortunately autodetect character encoding is broken since notepad++ version 7.6.
your file will be detected incorrectly as iso 8859-9 instead of utf-8, messing up your data, as long as you have autodetect character encoding enabled.

**workaround:**
currently, until a fix is rolled out, it is necessary to:
* go to `settings > preferences > misc` and **disable** `auto detect character encoding` as seen at the screenshot below.

![settings - auto detect character encoding](https://i.imgur.com/9SZ5ufh.png)

* then go to `settings > preferences > new document` and set your preferred default encoding to e.g utf-8, as seen at the screenshot below.

![Imgur](https://i.imgur.com/AarYeSx.png)

after doing that, please close and reopen your `notepad++_.ini_bug.txt` file.

your content will now be:
```
[test]
item=Boots(5m)¦Helmet(100m)
```
instead of the erroneous previous:
```
[test]
item=Boots(5m)Â¦Helmet(100m)
```

**note:**
i have attached your file below, for further internal testing purposes and convenience:

[eng, utf-8, detected as iso 8859-9 (#5440 notepad++_.ini_bug).txt](https://github.com/notepad-plus-plus/notepad-plus-plus/files/2985348/eng.utf-8.detected.as.iso.8859-9.5440.notepad%2B%2B_.ini_bug.txt)

**Comment by westor7** [comment]
Thank you @MetaChuh , i will wait until an official fix in next update.

**Comment by MetaChuh** [comment]
you are welcome @westor7 

please keep autodetect character encoding disabled, and set your preferred encoding to e.g. utf-8 or ansi manually, until you see the auto detection fix at the notepad++ change.log.

the fix might take some more time.
(we have not yet received any note, that this issue can be fixed before releasing the next update)

**Comment by MetaChuh** [comment]
@westor7
the fix will be available with the next 7.6.5. release.
we have re-tested your specific sample file, and it is now working as expected.
